docs/Installing.asciidoc

```diff
@@ -302,19 +302,28 @@ zypper dup --from devel_openQA_Leap --allow-vendor-change
 
 
 === Installation
-You can install the main openQA server package using these commands.
+
+==== Preparations on SLE
+
+On SLE some repositories have to be added.
```

Comment:
I'd phrase it more distinctly from adding repositories (which probably needs to be done in *addition*):
```suggestion
On SLE certain modules have to be enabled.
```